packages/account/src/connectors/types/fuel-storage.ts

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
I think we already have a abstraction for storage inside the account used by the vault-manager and we could merge both.